templates/article_page.hbs

```diff
@@ -233,15 +223,39 @@
                           </ul>
 
                           <div class="community-badge-container-achievements">
-                            {{#each author.badges}}
-                              {{#is category_slug "achievements"}}
-                                {{#if icon_url}}
-                                  <div class="community-badge community-badge-achievements">
-                                    <img src="{{icon_url}}" alt="{{name}}" title="{{name}} | {{description}}" aria-label="{{name}}" />
-                                  </div>
-                                {{/if}}
-                              {{/is}}
+                            {{#each (
+                              slice
+                                (
+                                  filter
+                                    (filter author.badges on="category_slug" equals="achievements")
+                                    on="icon_url"
+                                    starts_with="https"
+                                )
+                                0
+                                4
+                              )
+                            }}
+                              <div class="community-badge community-badge-achievements">
+                                <img src="{{icon_url}}" alt="{{name}}" title="{{name}} | {{description}}" />
```

Comment:
@zendesk/vikings should we have a translatable string for something like "Badge: {{name}}`. Otherwise, it might not be apparent the image is a user badge.

Reply:
I think it's a good idea to use an aria label.